src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml

```diff
@@ -22,7 +22,7 @@
         </li>
         <li>
             <p>Scan the QR Code or enter this key <kbd>@Model.SharedKey</kbd> into your two factor authenticator app. Spaces and casing do not matter.</p>
-            <div class="alert alert-info">To enable QR code generation please read our <a href="https://go.microsoft.com/fwlink/?Linkid=852423">documentation</a>.</div>
+            <div class="alert alert-info">To enable QR code generation, read our <a href="https://go.microsoft.com/fwlink/?Linkid=852423">documentation</a>.</div>
```

Comment:
For accessibility, link text "documentation" is not very descriptive; you could rearrange the text:
```suggestion
            <div class="alert alert-info">Learn how to <a href="https://go.microsoft.com/fwlink/?Linkid=852423">enable QR code generation</a>.</div>
```